### ISSUE TITLE
Ritual persists

### ISSUE THREAD

**Issue by n0yes** [HellFirePvP/AstralSorcery#481]
While playing on DW20 1.3.0 pack I had Bootes ritual running till I filled my drawers then I tried to disable it with redstone - which didn't work (I just randomly put lever near the pedestal), then I removed the crystal - which also didn't work so I removed the ritual pedestal block --> but the ritual still goes on and items are accumulating on the ground. Now I removed animals to prevent dangerous item spam.

**Comment by HellFirePvP** [comment]
Sounds like the network is corrupted and missing its parent block. Did you ever move the ritual by suspicious means? Anything that can move blocks without breaking them like Frames? Don't do that with any AS tile. It's blacklisted in most mechanisms by now, Frames seems to be the exception.

Stop the server/SP world, in the config set 'performNetworkIntegrityCheck' to true, startup the server/SP world, enter the world the ritual is in, let the integrity check finish, stop the server/SP world and set the config option to false again. Next startup it should vanish.

**Comment by n0yes** [comment]
Hmm. I didn't use Frames, I did use moving wand to place blocks probably (or swapping wand) and I did change setups few times because this is my 1st time doing AS and I'm just learning the ropes. So everything is possible. I had a collector crystal enhancing the ritual, then I removed it ... I tried to block the sunlight with piston (because I have no clue what's the proper way to disable the ritual when not needed). I did all sorts of shenanigans.

I am on a server though and will have to ask admin to fix this :/
